src/components/navigation-header/index.tsx

```diff
@@ -48,7 +46,7 @@ const NavigationHeader = () => {
       <LeftSideHeaderContent />
       <div className={s.rightSide}>
         <GiveFeedbackButton />
-        {shouldShowMenuButton && <SidebarMenuButton />}
```

Comment:
We can't use the viewport data to conditionally render components like this if it's markup that gets SSR'd, otherwise we get hydration mismatches. I shifted this to use CSS.

Reply:
While we're here... can we update the name from `SidebarMenuButton` to `MobileMenuButton`?

Reply:
Sure thing